src/Redux/api.tsx

```diff
@@ -763,19 +772,28 @@ const routes = {
     path: "/api/v1/shift/{id}/comment/",
     method: "POST",
   },
+
   // Notifications
   getNotifications: {
     path: "/api/v1/notification/",
+    method: "GET",
+    TRes: Type<NotificationResponse>(),
```

Comment:
@adriansliva for responses that are paginated, it'd be cleaner if it's wrapped with `PaginatedResponse`

```suggestion
    TRes: Type<PaginatedResponse<NotificationData>>(),
```